scalableminds/webknossos
# Relocating to dashboard after importing a dataset

This PR changes the redirect after importing a dataset to the dashboard and not just goes back in history as it is done previously. 

### URL of deployed dev instance (used for testing):
- [https://redirecttodatasetlistafterimport.webknossos.xyz](https://redirecttodatasetlistafterimport.webknossos.xyz)
### Steps to test:
- Upload a dataset and then configure the import settings and save or cancel the process.
- Canceling will go back in history which should mean the upload page will be opened
- Saving will should take you back to the dashboard

### Issues:
- fixes  #3113

------
- ~~~[ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~~
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
- ~~~[ ] Updated [documentation](../blob/master/docs) if applicable~~~
- ~~~[ ] Needs datastore update after deployment~~~
